src/main/java/in/org/projecteka/hiu/GatewayCaller.java

```diff
@@ -0,0 +1,18 @@
+package in.org.projecteka.hiu;
+
+import in.org.projecteka.hiu.user.Role;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.List;
+
+@AllArgsConstructor
+@Getter
+@Builder
+public class GatewayCaller {
+    private String username;
+    private Boolean isServiceAccount;
+    List<Role> roles;
+    private boolean verified;
+}
```

Comment:
As it is a Service caller.. U can rename GatewayCaller as Service Caller
As the service_urls only called by other systems, isServiceAccount will always be true.. So u can remove it.
As u are passing true for verified always.. u can remove it too
Please refer consent_manager serviceCaller model.